.github/workflows/ci_pr-gatekeeper.yml

```diff
@@ -12,7 +12,6 @@ on:
         review_requested,
         review_request_removed
       ]
```

Comment:
Actually, I think we should do this:

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-when-a-pull-request-is-approved